test/test_t2star.py

```diff
@@ -114,11 +115,13 @@ def run(self, qobj):
                         )
 
                     if op.name == "measure":
-                        # measure in |+> basis
+                        # theoretically we measure in |+> basis
+                        # in practice we translate our measurement to 0/1 basis
+                        prob1[qubit] = 1.0 - prob_plus[qubit]
```

Comment:
No need to maintain `prob1` and `prob_plus` at the same time. Initially set `prob_plus = np.ones(nqubits)`, and continue to work with it:
```
(1 - prob_plus[qubit]) * (1 - r10[qubit]) + prob_plus[qubit] * r01[qubit]
```
And you can remove the comment about theoretically measuring in |+> basis etc.

Reply:
I accepted your suggestion